src/ostorlab/runtimes/local/runtime.py

```diff
@@ -434,7 +434,7 @@ def _start_agent(
 
     @tenacity.retry(
         stop=tenacity.stop_after_attempt(20),
-        wait=tenacity.wait_exponential(multiplier=1, max=12),
+        wait=tenacity.wait_fixed(0.5),
```

Comment:
We may need to increase the `stop_after_attempt` though..

Reply:
Good point, I will check in the case of multiple agents.